{"uuid": "a3304dd5-17ae-49ad-ba2a-e914f865235e", "children": ["eabaae31-007c-492a-92cc-0d71c64f64ed", "54a103c1-3dfc-4384-9c98-85c25102e183", "8b495f52-2805-447d-8b94-3756dae634eb", "5885ddcf-725c-44b0-9ada-56ec4b746fb5"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=123.0.6312.86)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7358A70C2+63090]\n\t(No symbol) [0x00007FF735812D12]\n\t(No symbol) [0x00007FF7356AEC65]\n\t(No symbol) [0x00007FF7356AAC1A]\n\t(No symbol) [0x00007FF73569D514]\n\t(No symbol) [0x00007FF73569E8F8]\n\t(No symbol) [0x00007FF73569D7FF]\n\t(No symbol) [0x00007FF73569D104]\n\t(No symbol) [0x00007FF73569D021]\n\t(No symbol) [0x00007FF73569B0A8]\n\t(No symbol) [0x00007FF73569B8BC]\n\t(No symbol) [0x00007FF7356B14AD]\n\t(No symbol) [0x00007FF735733D67]\n\t(No symbol) [0x00007FF735716FDA]\n\t(No symbol) [0x00007FF735733412]\n\t(No symbol) [0x00007FF735716D83]\n\t(No symbol) [0x00007FF7356E83A8]\n\t(No symbol) [0x00007FF7356E9441]\n\tGetHandleVerifier [0x00007FF735CA262D+4238301]\n\tGetHandleVerifier [0x00007FF735CDF78D+4488509]\n\tGetHandleVerifier [0x00007FF735CD7A6F+4456479]\n\tGetHandleVerifier [0x00007FF735980606+953270]\n\t(No symbol) [0x00007FF73581E5DF]\n\t(No symbol) [0x00007FF7358192B4]\n\t(No symbol) [0x00007FF7358193EB]\n\t(No symbol) [0x00007FF735809C24]\n\tBaseThreadInitThunk [0x00007FFC96F7257D+29]\n\tRtlUserThreadStart [0x00007FFC9920AA58+40]\n\n", "trace": "  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\PycharmProjects\\seleniumFramework\\tests\\conftest.py\", line 48, in setup\n    driver.get(\"https://rahulshettyacademy.com/angularpractice/\")\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1711968800106, "stop": 1711968801795}, {"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=123.0.6312.86)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7358A70C2+63090]\n\t(No symbol) [0x00007FF735812D12]\n\t(No symbol) [0x00007FF7356AEC65]\n\t(No symbol) [0x00007FF7356AAC1A]\n\t(No symbol) [0x00007FF73569D514]\n\t(No symbol) [0x00007FF73569E8F8]\n\t(No symbol) [0x00007FF73569D7FF]\n\t(No symbol) [0x00007FF73569D104]\n\t(No symbol) [0x00007FF73569D021]\n\t(No symbol) [0x00007FF73569B0A8]\n\t(No symbol) [0x00007FF73569B8BC]\n\t(No symbol) [0x00007FF7356B14AD]\n\t(No symbol) [0x00007FF735733D67]\n\t(No symbol) [0x00007FF735716FDA]\n\t(No symbol) [0x00007FF735733412]\n\t(No symbol) [0x00007FF735716D83]\n\t(No symbol) [0x00007FF7356E83A8]\n\t(No symbol) [0x00007FF7356E9441]\n\tGetHandleVerifier [0x00007FF735CA262D+4238301]\n\tGetHandleVerifier [0x00007FF735CDF78D+4488509]\n\tGetHandleVerifier [0x00007FF735CD7A6F+4456479]\n\tGetHandleVerifier [0x00007FF735980606+953270]\n\t(No symbol) [0x00007FF73581E5DF]\n\t(No symbol) [0x00007FF7358192B4]\n\t(No symbol) [0x00007FF7358193EB]\n\t(No symbol) [0x00007FF735809C24]\n\tBaseThreadInitThunk [0x00007FFC96F7257D+29]\n\tRtlUserThreadStart [0x00007FFC9920AA58+40]\n\n", "trace": "  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\PycharmProjects\\seleniumFramework\\tests\\conftest.py\", line 48, in setup\n    driver.get(\"https://rahulshettyacademy.com/angularpractice/\")\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1711968801837, "stop": 1711968803558}, {"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=123.0.6312.86)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7358A70C2+63090]\n\t(No symbol) [0x00007FF735812D12]\n\t(No symbol) [0x00007FF7356AEC65]\n\t(No symbol) [0x00007FF7356AAC1A]\n\t(No symbol) [0x00007FF73569D514]\n\t(No symbol) [0x00007FF73569E8F8]\n\t(No symbol) [0x00007FF73569D7FF]\n\t(No symbol) [0x00007FF73569D104]\n\t(No symbol) [0x00007FF73569D021]\n\t(No symbol) [0x00007FF73569B0A8]\n\t(No symbol) [0x00007FF73569B8BC]\n\t(No symbol) [0x00007FF7356B14AD]\n\t(No symbol) [0x00007FF735733D67]\n\t(No symbol) [0x00007FF735716FDA]\n\t(No symbol) [0x00007FF735733412]\n\t(No symbol) [0x00007FF735716D83]\n\t(No symbol) [0x00007FF7356E83A8]\n\t(No symbol) [0x00007FF7356E9441]\n\tGetHandleVerifier [0x00007FF735CA262D+4238301]\n\tGetHandleVerifier [0x00007FF735CDF78D+4488509]\n\tGetHandleVerifier [0x00007FF735CD7A6F+4456479]\n\tGetHandleVerifier [0x00007FF735980606+953270]\n\t(No symbol) [0x00007FF73581E5DF]\n\t(No symbol) [0x00007FF7358192B4]\n\t(No symbol) [0x00007FF7358193EB]\n\t(No symbol) [0x00007FF735809C24]\n\tBaseThreadInitThunk [0x00007FFC96F7257D+29]\n\tRtlUserThreadStart [0x00007FFC9920AA58+40]\n\n", "trace": "  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\PycharmProjects\\seleniumFramework\\tests\\conftest.py\", line 48, in setup\n    driver.get(\"https://rahulshettyacademy.com/angularpractice/\")\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1711968803607, "stop": 1711968805386}, {"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=123.0.6312.86)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7358A70C2+63090]\n\t(No symbol) [0x00007FF735812D12]\n\t(No symbol) [0x00007FF7356AEC65]\n\t(No symbol) [0x00007FF7356AAC1A]\n\t(No symbol) [0x00007FF73569D514]\n\t(No symbol) [0x00007FF73569E8F8]\n\t(No symbol) [0x00007FF73569D7FF]\n\t(No symbol) [0x00007FF73569D104]\n\t(No symbol) [0x00007FF73569D021]\n\t(No symbol) [0x00007FF73569B0A8]\n\t(No symbol) [0x00007FF73569B8BC]\n\t(No symbol) [0x00007FF7356B14AD]\n\t(No symbol) [0x00007FF735733D67]\n\t(No symbol) [0x00007FF735716FDA]\n\t(No symbol) [0x00007FF735733412]\n\t(No symbol) [0x00007FF735716D83]\n\t(No symbol) [0x00007FF7356E83A8]\n\t(No symbol) [0x00007FF7356E9441]\n\tGetHandleVerifier [0x00007FF735CA262D+4238301]\n\tGetHandleVerifier [0x00007FF735CDF78D+4488509]\n\tGetHandleVerifier [0x00007FF735CD7A6F+4456479]\n\tGetHandleVerifier [0x00007FF735980606+953270]\n\t(No symbol) [0x00007FF73581E5DF]\n\t(No symbol) [0x00007FF7358192B4]\n\t(No symbol) [0x00007FF7358193EB]\n\t(No symbol) [0x00007FF735809C24]\n\tBaseThreadInitThunk [0x00007FFC96F7257D+29]\n\tRtlUserThreadStart [0x00007FFC9920AA58+40]\n\n", "trace": "  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\owusu\\PycharmProjects\\seleniumFramework\\tests\\conftest.py\", line 48, in setup\n    driver.get(\"https://rahulshettyacademy.com/angularpractice/\")\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\owusu\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1711968809482, "stop": 1711968811142}], "start": 1711968809482, "stop": 1711968811184}